lib/org/chromium/webidl/IDLFragmentExtractor.js

```diff
@@ -21,96 +22,179 @@ foam.CLASS({
       },
     },
     {
+      class: 'Array',
+      of: 'String',
+      documentation: 'Array of IDL Fragments found in the parsed HTML file.',
       name: 'idlFragments',
-      // description: 'all idlFragments found in the parsed HTML file',
+    },
+    {
+      name: 'tags_',
+      documentation: 'Array of tags extracted from file to be processed.',
+      expression: function(file) {
+        var results = this.lexer_.parseString(file.contents).value;
+        if (!results)
+          throw new Error("IDLFragmentExtractor: IDL Parse was not successful.");
```

Comment:
Maybe after the colon `HTML Lexer failed to parse document` + maybe include the URL in  the message?